templates/project/cargo-generate.toml

```diff
@@ -8,3 +8,11 @@ type = "string"
 prompt = "MCU"
 choices = ["esp32"]
 default = "esp32" 
+
+[placeholders.camera]
+type = "bool"
+prompt = "include camera?"
```

Comment:
Can we make this just prompt just slightly less terse?

Reply:
`include camera module`? fine with any suggestions